pkg/plugins/external/edit.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package external
 
 import (
+	"github.com/spf13/pflag"
+
```

Comment:
@everettraven,

We need to squash the commits for Kubebuilder as well. 
Could you please squash them?